CMakeLists.txt

```diff
@@ -67,12 +67,12 @@ IF (BUILD_FUZZER)
     ENDIF()
     ADD_EXECUTABLE(quicly-fuzzer-packet fuzz/packet.cc ${PICOTLS_OPENSSL_FILES})
     SET(LIB_FUZZER "${CMAKE_CURRENT_BINARY_DIR}/libFuzzer.a")
```

Comment:
Te me it seems that we are switching to using `-fsanitize=fuzzer`,  and that this variable is no longer used.

Does the switch mean that we can remove:
* this line,
* and also the "libFuzzer" custom target that download and builds the fuzzing library as well as the files being used for that purpose?

Reply:
My assumption was incorrect, so I think I'll need to make changes. Initially I wasn't building with -DBUILD_FUZZER=on cmake option, but based on your comment (and looking further), that is how it should be done. With that said, it appears that in fuzz/packet.cc there is the use of an identifier "quicly_default_context", which at the time the packet.cc was extended, that identifier was still being used. The most recent version of quicly/defaults.h doesn't have that declaration any longer. This means that fuzz/packet.cc has to be modified, so this will require further discussion.